deploy/10_layer_zero_module.ts

```diff
@@ -109,13 +195,55 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
 
   const holograph = await hre.ethers.getContract('Holograph', deployer);
 
+  const futureOptimismGasPriceOracleAddress = await genesisDeriveFutureAddress(
```

Comment:
This needs to be deployed via Super Cold Storage right?

Reply:
For `minnet` and `testnet` yes, but for `develop` we can deploy this with deployer wallet.

Reply:
cool that's what i thought